STUDIO/org.openl.rules.jackson/src/org/openl/rules/serialization/ProjectJacksonObjectMapperFactoryBean.java

```diff
@@ -547,7 +555,19 @@ public boolean isFailOnEmptyBeans() {
         return delegate.isFailOnEmptyBeans();
     }
 
+    public String getTypingPropertyName() {
+        return delegate.getTypingPropertyName();
+    }
+
     public void setTypingPropertyName(String typingPropertyName) {
         delegate.setTypingPropertyName(typingPropertyName);
     }
+
+    public boolean isDeduction() {
+        return delegate.isDeduction();
+    }
+
+    public void setDeduction(boolean deduction) {

```

Comment:
Can be it an Enum value of `JsonTypeInfo.Id` instead of boolean? Like it did for `setSerializationInclusion()`